pythonforandroid/recipes/png/__init__.py

```diff
@@ -3,8 +3,8 @@
 
 class PngRecipe(NDKRecipe):
     name = 'png'
-    version = '1.6.15'
-    url = 'https://github.com/julienr/libpng-android/archive/{version}.zip'
+    version = '1.6.29'
+    url = 'https://github.com/julienr/libpng-android/archive/master.zip'
```

Comment:
Oops, I guess you hardcoded the branch/tag by mistake right?

Reply:
oh wow didn't notice that one :open_mouth: now I feel blind. yeah, would be nice to know if that is an accident or not

Reply:
Unfortunately, there is no error...well...I forgot to notify it (**my apologies for that...I completely forgot about it...**)...

Well...the thing is that the author of the github's repo never released/tagged this version, despite He performed the necessary changes in master branch and notify about it in the README file...I thing that we should try to move it from that repo to the mainline repo, plus considering that the last commit is more than a year old...mmm...I have bad feelings about that...

Anyway...maybe we should target a commit instead of a version, this way we wouldn't hardcode the url...I still think that we should try to move it to mainline but maybe would be better to do it in another pr (this way we could solve the current Pillow/pil issues, and move the png recipe to the mainline repository when we can).

@JonasT and @AndreMiras, what do you think?

Reply:
@opacam yea hardcoding a commit seems like the best idea for now :+1: